src/views/Tournament/Tournament.tsx

```diff
@@ -1745,13 +1745,18 @@ export function Tournament(): JSX.Element {
                             </div>
                         )}
                         {(sorted_players.length > 0 || null) && (
-                            <Card>
-                                {sorted_players.map((player) => (
-                                    <div key={player.id}>
-                                        <Player icon user={player} />
-                                    </div>
-                                ))}
-                            </Card>
+                            <>
+                                <Card>
+                                    {sorted_players.map((player) => (
+                                        <div key={player.id}>
+                                            <Player icon user={player} />
+                                        </div>
+                                    ))}
+                                </Card>
+                                <div className="player-count">
+                                    Number of players: {sorted_players.length}
```

Comment:
Need to translate this string, probably something like `{interpolate(_("Number of players: {{num_players}}"), { num_players: sorted_players.length})}`

Reply:
Done!